ruby/deepgram-audio-summary/README.md

```diff
@@ -0,0 +1,67 @@
+# 🔉📃 Transcribe Audio with Deepgram
+
+A Ruby Cloud Function for transcribing audio number using [Deepgram API](https://deepgram.com/).
+
+_Example input:_
+
+```json
+{
+    "fileUrl": "https://static.deepgram.com/examples/interview_speech-analytics.wav"
+}
+```
+
+_Example output:_
+
+
+```json
+{
+    "success": true,
+    "deepgramData": [
+        {
+            "summary": "another big problem in the speech analytics space. When customers first bring the software on is that they they are blown away by the fact that an engine can monitor hundreds of Kpis.",
+            "start_word": 0,
+            "end_word": 228
+        }
+    ]
+}
+```
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud function:
+
+- **DEEPGRAM_API_KEY** - Deepgram API Key
+
+ℹ️ _Create your Deepgram API key at console.deepgram.com_
+
+## 🚀 Deployment
+
+1. Clone this repository, and enter this function folder:
+
+```bash
+git clone https://github.com/open-runtimes/examples.git && cd examples
+cd ruby/deepgram-transcribe-audio
```

Comment:
cd ruby/deepgram-transcribe-audio is wrong here

Reply:
fixed